db/seeds.rb

```diff
@@ -88,17 +313,22 @@ def weighed_number(weights)
   ranges.find{ |p, _| p > u }.last
 end
 
-200.times do |order|
-  t1 = Time.parse("2012-11-16 12:00:00")
-  t2 = Time.parse("2018-09-15 12:00:00")
-  created = Order.create(user_id: User.all.sample.id, status: weighed_number({0 => 0.2, 1 => 0.1, 2 => 0.1, 3 => 0.6 }), created_at: rand(t1..t2), updated_at: Time.now)
-  random = weighed_number({1 => 0.2, 2 => 0.2, 3 => 0.2, 4 => 0.2, 5 => 0.2 })
-  random.times do |add_item|
-    created.items << Item.all.sample
+users = User.all
+random_order_count = rand(10..20)
+users.each do |user|
+  random_order_count.times do |order|
+    t1 = Time.parse("2012-11-16 12:00:00")
+    t2 = Time.parse("2018-09-15 12:00:00")
+    created = Order.create(user_id: user.id, status: weighed_number({0 => 0.2, 1 => 0.1, 2 => 0.1, 3 => 0.6 }), created_at: rand(t1..t2), updated_at: Time.now)
+    random = weighed_number({1 => 0.2, 2 => 0.2, 3 => 0.2, 4 => 0.2, 5 => 0.2 })
+    random.times do |add_item|
+      created.items << Item.all.sample
+    end
+    puts "Created order #{created.id} for #{created.user.name}"
   end
-  puts "Created order #{created.id} for #{created.user.name}"
 end
 
 puts "Creating the admin"
-# admin role still needs to be added
+
 User.create(name: "admin", username: "admin", password: "admin", role: "admin")
+User.create(name: "Josh", username: "josh@turing.io", password: "password")
```

Comment:
For production we might want to remove these and generate in the console so don't have users with public login info on Github